spec.emu

```diff
@@ -136,189 +175,307 @@ location: https://github.com/tc39/proposal-decimal/
-        1. Assert: *_c_* is in « -1, 0, 1 ».
-        1. Return *_c_*.
+        1. Let _O_ be the **this** value.
+        1. If [_O_][[isNaN]] is *true*, then
+          1. If [*_x_*][[isNaN]] is *true*, return *true*.
+          1. Return *false*.
+        1. If [*_x_*][[isNaN]] is *true*, return *false*.
+        1. Let _d1_ be _O_.[[Decimal128Data]].
+        1. Let _d2_ be _x_.[[Decimal128Data]].
+        1. If the mathematical value of of _d1_ is less than the mathematical value of _d2_, return *-1*<sub>𝔽</sub>.
+        1. If the mathematical value of of _d2_ is less than the mathematical value of _d1_, return *1*<sub>𝔽</sub>.
+        1. Let _v1_ be _d1_[1].
+        1. Let _v2_ be _d2_[1].
+        1. If the mathematical value of _v1_ is less than the mathematical value of _v2_, return *-1*<sub>𝔽</sub>.
+        1. If the mathematical value of _v2_ is less than the mathematical value of _v1_, return *1*<sub>𝔽</sub>.
+        1. Return *+0*<sub>𝔽</sub>.
       </emu-alg>
       <emu-note>
         <p>This operation compares Decimal128 values as digit strings. Thus, for example, 1.2 and 1.20 are considered distinct values for this operation. To compare by mathematical value, see <emu-xref href="#sec-decimal.prototype.lessthan">lessThan</emu-xref> and <emu-xref href="#sec-decimal.prototype.equals">equals</emu-xref>.</p>
       </emu-note>
     </emu-clause>
     <emu-clause id="sec-decimal-method-round">
       <h1>Decimal128.prototype.round ( _numFractionalDigits_ [ , _mode_ ] )</h1>
-      <p>Rounds *this* value according to the specified rounding mode, returning a fresh result. The result will have at most _numFractionalDigits_ after the decimal point.</p>
+      <p>This method performs the following steps when called:</p>
       <emu-note>
         <p>If _mode_ is neither *undefined* nor a String, throw a *TypeError* exception.</p>
         <p>If _mode_ is not a <emu-xref href="#dfn-decimal128-rounding-mode">rounding mode</emu-xref>, throw a *RangeError* exception.</p>
       </emu-note>
     </emu-clause>
     <emu-clause id="sec-decimal.prototype.tostring">
       <h1>Decimal128.prototype.toString ( [ _options_ ] )</h1>
-      <p>Renders *this* value as a signed decimal string. _options_, is defined, is supposed to be a plain object. If _options_ has the property *numDecimal128Digits* and the value of that property is a non-negative Number, the return value will be a string that has exactly that many digits after the decimal point, possibly with the addition of trailing zeros.</p>
+      <p>This method performs the following steps when called:</p>
       <emu-alg>
         1. Let _roundingMode_ be [[_options_]][roundingMode]].
-        1. If _roundingMode_ is not a rounding mode, then
+        1. If _roundingMode_ is not a <emu-xref href="#dfn-decimal128-rounding-mode">rounding mode</emu-xref>, then
           1. set _roundingMode_ to *"roundTiesToEven"*.
```

Comment:
Not throw? Otherwise "Set" with a capital letter

Reply:
We could throw. I've adopted that suggestion.